mkosi

```diff
@@ -429,7 +433,7 @@ def install_debian_or_ubuntu(args, workspace, run_build_script, mirror):
     cmdline = ["debootstrap",
                "--verbose",
                "--variant=minbase",
-               "--include=systemd",
+               "--include=systemd,usrmerge",
```

Comment:
@stefan-it could you check out https://github.com/fsateler/mkosi/commit/3656ae181c052a5f0b22456d364ed9dc4c40854b ?